description_templates/golden_test.go

```diff
@@ -49,16 +60,52 @@ func TestGenerateGolden(t *testing.T) {
 		PricePerUnit: 0.000000746,
 		Discount:     0.33,
 		Total:        43.962005025946798,
+		SubItems: map[string]invoice.SubItem{
+			"appuio_cloud_memory_subquery_memory_request": {
+				Description: "Memory request aggregated by namespace",
+				QueryName:   "appuio_cloud_memory_subquery_memory_request",
+				Quantity:    34923234.04433424,
+				QuantityMin: 2.251,
+				QuantityAvg: 42.2,
+				QuantityMax: 9001.1,
+				Unit:        "TPS",
+			},
+			"appuio_cloud_memory_subquery_cpu_request": {
+				Description: "CPU requests in memory request equivalent",
+				QueryName:   "appuio_cloud_memory_subquery_cpu_request",
+				Quantity:    44323235.04444221,
+				QuantityMin: 2.251,
+				QuantityAvg: 133.7,
+				QuantityMax: 9001.1,
+				Unit:        "TPS",
+			},
+		},
 	}
 
 	for _, key := range sourceKeys {
 		t.Run(key, func(t *testing.T) {
 			item := baseItem
 			item.ProductRef.Source = key
-			rendered, err := r.RenderItemDescription(context.Background(), item)
+			if key == "appuio_cloud_memory:c-appuio-exoscale-ch-gva-2-0" {
```

Comment:
I'd prefer this to be it's own test and not inside a golden test. It's a bit too magic here for my taste.

Reply:
Fair enough. That was something I realized would fail after writing and wanted to quickly add a test. I'll move it to a dedicated test.